cocos/Android.mk

```diff
@@ -165,7 +165,7 @@ LOCAL_WHOLE_STATIC_LIBRARIES := cocos2dxandroid_static
 LOCAL_WHOLE_STATIC_LIBRARIES += cpufeatures
 
 # define the macro to compile through support/zip_support/ioapi.c
-LOCAL_CFLAGS := -DUSE_FILE32API -fexceptions -DHAVE_INSPECTOR
```

Comment:
`HAVE_INSPECTOR` have defined in `config.hpp`, like this
```
#if defined(COCOS2D_DEBUG) && COCOS2D_DEBUG > 0
#define SE_ENABLE_INSPECTOR 1
#define SE_DEBUG 2
#define HAVE_INSPECTOR 1
#else
#define SE_ENABLE_INSPECTOR 0
#define SE_DEBUG 0
#define HAVE_INSPECTOR 0
#endif
```
it cause many re-defined errors when building native code.